shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/plants/presentation/detail/components/PlantDetailScreen.kt

```diff
@@ -22,10 +71,304 @@ fun PlantDetailScreen(
+            )
+        }
+    }
+}
+
+@Composable
+private fun PlantDetails(
+    plant: Plant,
+    quantity: Int,
+    onEvent: (PlantDetailEvent) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Card(
+        shape = RoundedCornerShape(topStart = 40.dp, topEnd = 40.dp),
+        elevation = CardDefaults.cardElevation(defaultElevation = 8.dp),
+        modifier = modifier
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(Color.White)
+                .padding(24.dp)
+        ) {
+            Box(
+                modifier = Modifier
+                    .height(4.dp)
+                    .width(40.dp)
+                    .clip(RoundedCornerShape(2.dp))
+                    .background(Color.Gray.copy(alpha = 0.8f))
+                    .align(Alignment.CenterHorizontally),
+            )
+
+            // PLANT NAME
+            Spacer(modifier = Modifier.height(24.dp))
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = plant.name,
+                    fontWeight = FontWeight.Bold,
+                    color = Color.Black,
+                    fontSize = 24.sp
+                )
+                Spacer(modifier = Modifier.weight(1.0f))
+                Text(
+                    text = plant.originalName,
+                    fontWeight = FontWeight.Bold,
+                    color = MaterialTheme.colorScheme.primary.copy(alpha = 0.6f),
+                    fontSize = 14.sp
+                )
+            }
+
+            // PLANT DESCRIPTION
+            Spacer(modifier = Modifier.height(24.dp))
+            Text(
+                text = plant.description,
+                color = Color.Gray,
+                fontSize = 14.sp
+            )
+
+            // PLANT DETAILS INFO
+            Spacer(modifier = Modifier.height(24.dp))
+            PlantExtraInfoRow(plant.details)
+
+            // PLANT QUANTITY
+            Spacer(modifier = Modifier.height(24.dp))
+            Row() {
+                QuantityController(
+                    value = quantity,
+                    onQuantityChanged = { quantity ->
+                        onEvent(PlantDetailEvent.OnQuantityChanged(quantity))
+                    }
+                )
+                Spacer(modifier = Modifier.weight(1.0f))
+                Button(
+                    modifier = Modifier
+                        .clip(RoundedCornerShape(24.dp))
+                        .background(MaterialTheme.colorScheme.primary),
+                    onClick = {
+                        onEvent(PlantDetailEvent.CheckoutPlant)
+                    }
+                ) {
+                    Text(
+                        text = UiText.StringRes(checkout_with_price, listOf(plant.price)).asString(),
```

Comment:
The text is not displayed correctly on iOS :)

Reply:
Fixed :)